spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/PackageController.java

```diff
@@ -108,13 +106,22 @@ public void packageDelete(@PathVariable("name") String name) {
 		this.packageMetadataService.deleteIfAllReleasesDeleted(name, PackageMetadataService.DEFAULT_RELEASE_ACTIVITY_CHECK);
 	}
 
+	@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Release not found")
+	@ExceptionHandler(ReleaseNotFoundException.class)
+	public void handleReleaseNotFoundException() {
+		// needed for server not to log 500 errors
+	}
+
+	@ResponseStatus(value = HttpStatus.PRECONDITION_FAILED, reason = "Package deletion error")
```

Comment:
I think that `412 Precondition Failed` is not a correct message status.
 Although the message name sound relevant, HTTP has very concrete semantics related to conditional requests  such as  `If-Unmodified-Since` or `If-None-Match`. (https://tools.ietf.org/html/rfc7232#section-4.2)
IMO we should use HTTP 409 (CONFLICT) instead